src/progress-bar/interfaces.ts

```diff
@@ -69,7 +69,43 @@ export interface ProgressBarProps extends BaseComponentProps {
   onResultButtonClick?: NonCancelableEventHandler;
 }
 
+interface PercentageProgressBarProps extends BaseProgressBarProps {
+  /**
+   * Specifies the progress type.
+   *
+   * @defaultValue 'percentage'
```

Comment:
Nice idea, but we parse public interface comments to generate our public API documentation. Please keep these comments limited to documentation strings.